app/exawind/exawind.cpp

```diff
@@ -157,7 +160,7 @@ int main(int argc, char** argv)
     for (int i = 0; i < num_nwsolvers; i++) {
         if (nalu_comms.at(i) != MPI_COMM_NULL)
             sim.register_solver<exawind::NaluWind>(
-                nalu_comms.at(i), nalu_inps.at(i), nalu_vars);
+                i + 1, nalu_comms.at(i), nalu_inps.at(i), nalu_vars);
```

Comment:
I actually kinda like numbering from 0. So I guess I would have just used `i` here instead of `i+1`. What do you think?

Reply:
I see your point about being consistent within the code. Currently identifiers are used only for output to the screen/file, and hence I picked a starting index of 1. That felt more intuitive to me while reading the timing for many nalu instances.

Reply:
yeah this totally a preference thing. I like 0-based but don't care that much.